src/reports/NPSP_Health_Check/Individual_Accounts_by_Account_Type.report

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Report xmlns="http://soap.sforce.com/2006/04/metadata">
+    <columns>
+        <field>USERS.NAME</field>
+    </columns>
+    <columns>
+        <field>ACCOUNT.NAME</field>
+    </columns>
+    <columns>
+        <field>TYPE</field>
+    </columns>
+    <columns>
+        <field>RATING</field>
+    </columns>
+    <columns>
+        <field>DUE_DATE</field>
+    </columns>
+    <columns>
+        <field>LAST_UPDATE</field>
+    </columns>
+    <columns>
+        <field>ADDRESS1_STATE</field>
+    </columns>
+    <description>Lists Accounts grouped by their system Account Type.  Details are hidden to avoid data limits.  Check the group you want to see details on and click the Drill Down button.</description>
```

Comment:
@jvarese I copied over the description from the old Individual Contacts by Account Type health check report.  This message is visible in the report selection list, in the Reports tab.

<img width="637" alt="screen shot 2016-07-18 at 8 13 11 pm" src="https://cloud.githubusercontent.com/assets/1058323/16949836/83a0aa7e-4d82-11e6-927a-1d9991d2e72f.png">